### ISSUE TITLE
Should we fork this repo to keep it updated?

### ISSUE THREAD

**Issue by WAZAAAAA0** [roguelike2d/TekkenBot#62]
The main author behind the TekkenBotPrime has become inactive with no one left to maintain it. Stuff happened in the meanwhile.

Tekken 7 has been patched multiple times, thus rendering the program outdated.

Someone made a commit to fix the memory_address override, which would allow people to more easily update the program by simply entering the correct addresses in the ini, no need to re-build anything or even get python, just in case the next maintainer goes inactive too: https://github.com/roguelike2d/TekkenBot/pull/55

The method to manually update addresses with CE is explained here and could be highlighted more in the next updated README.md: https://github.com/roguelike2d/TekkenBot/issues/15

@Pobega explained how to properly run the source with python, should be explained in the README.md: https://github.com/roguelike2d/TekkenBot/issues/52

The auto-update function should also get properly migrated or removed.

Should we do this? Who should do this?

**Comment by sweetclimusic** [comment]
Yes you should. The easiest thing it to start pulling from @PRoosta fork of tekkenbot. Then any changes from that fork can always come back to this one.

**Comment by TehLeico** [comment]
Well, Tekken just got updated right now, it's the Geese patch, of course, TekkenBotPrime doesn't work anymore, this is the best tool that Tekken ever had, it would be a shame if it is abandoned
